GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -33,6 +33,7 @@ public class FileSystemCallbacks : IDisposable, IHeartBeatMetadataProvider
         private GVFSContext context;
         private GVFSGitObjects gitObjects;
         private ModifiedPathsDatabase modifiedPaths;
+        private ConcurrentHashSet<string> newPaths;

```

Comment:
This name isn't super clear. Are the "paths" new? Or are these the paths of newly created files?

I would rename this to `newlyCreatedFiles`, or even better though more verbose, `newlyCreatedFilesAndDirectories`

Reply:
I'll go with `newlyCreatedFileAndFolderPaths` if that works?